### PR TITLE
Updated scripts for 4.x

### PR DESCRIPTION
**Pull request by wewang58**
Passed in my local host, @openshift/devexp-qe pls help to review it, thanks

### CODE REVIEW

**Comment by akostadinov** [comment]
lgtm

File: features/cli/resources.feature [hunk 1]
```diff
@@ -14,10 +14,10 @@ Feature: resouces related scenarios
...
       | Name:\\s+database            |
       | Selector:\\s+name=database   |
-    When I run the :export client command with:
-      | resource     | svc           |
-      | name         | database      |
-      | o            | yaml          |
+    When I run the :get client command with:
+      | resource      | svc      |
+      | resource_name | database |
+      | output        | yaml     |
```
Comment (akostadinov) [inline features/cli/resources.feature]: I wonder if using the `--export` option will be better here.

Comment (wewang58) [reply features/cli/resources.feature]: It's not support export command in 4.x now

File: features/cli/oc_set_build_hook.feature [hunk 2]
```diff
@@ -36,18 +36,4 @@ Feature: oc_set_build_hook
     When I run the :start_build client command with:
       | buildconfig | rails-postgresql-example |
     Then the step should succeed
-    Given the "rails-postgresql-example-3" build completes
-    When I run the :set_build_hook client command with:
```
Comment (xiuwang) [inline features/cli/oc_set_build_hook.feature]: If the set hook feature is removed from 4.x?

Comment (xiuwang) [inline features/cli/oc_set_build_hook.feature]: If the set hook feature is removed from 4.x?

File: features/cli/resources.feature (hunk 1 above)
Comment (akostadinov) [reply features/cli/resources.feature]:
```
🐚 oc get -h | grep export
      --export=false: If true, use 'export' for the resources.  Exported resources are stripped of cluster-specific information.
```

Comment (wewang58) [reply features/cli/resources.feature]: Oh, I misunderstood  you, I thought "oc export xxx", I will update it later.

File: features/cli/oc_set_build_hook.feature (hunk 2 above)
Comment (wewang58) [reply features/cli/oc_set_build_hook.feature]: No, just set hook commond did not support with args options in 4.x. so remove the part. there's a bug: https://bugzilla.redhat.com/show_bug.cgi?id=1670000

File: features/cli/resources.feature (hunk 1 above)
Comment (wewang58) [reply features/cli/resources.feature]: @akostadinov  Updated it with export option, pls review it, thanks.

File: features/cli/oc_set_build_hook.feature (hunk 2 above)
Comment (akostadinov) [reply features/cli/oc_set_build_hook.feature]: bug shows verified. Do we still need to remove those?

File: features/cli/resources.feature (hunk 1 above)
Comment (akostadinov) [reply features/cli/resources.feature]: Thank you!

File: features/cli/oc_set_build_hook.feature (hunk 2 above)
Comment (wewang58) [reply features/cli/oc_set_build_hook.feature]: Hi @akostadinov you can see https://github.com/openshift/origin/pull/21880/files#diff-77d633c62fb7fa312b6e7edff1743c7cL2942 in pr https://github.com/openshift/origin/pull/21389/files, codes already remove it, so scripts removed it too.

**Comment by wewang58** [comment]
@openshift/devexp-qe  I think it can be merge now.

**Pull request merged**